docker/docker-compose.yaml

```
@@ -55,66 +58,127 @@ services:
-      use_sim_time:=${USE_SIM_TIME:-true}
-      vehicle_id:=${VEHICLE_ID:-default}
-      vehicle_model:=${VEHICLE_MODEL:-sample_vehicle}
-    restart: unless-stopped
+      lateral_controller_mode:=${LATERAL_CONTROLLER_MODE}
+      longitudinal_controller_mode:=${LONGITUDINAL_CONTROLLER_MODE}
+      module_preset:=${CONTROL_MODULE_PRESET}
+      use_sim_time:=${USE_SIM_TIME}
+      vehicle_id:=${VEHICLE_ID}
+      vehicle_model:=${VEHICLE_MODEL}
+    restart: on-failure
 
   vehicle:
     image: ghcr.io/autowarefoundation/autoware:universe-vehicle-system
     container_name: autoware-vehicle
     pid: service:map
+    ipc: host
     init: true
     environment:
-      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID:-0}
+      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
     networks:
       - autoware-net
     depends_on:
       - map
     command: >
       ros2 launch tier4_vehicle_launch vehicle.launch.xml
-      launch_vehicle_interface:=${LAUNCH_VEHICLE_INTERFACE:-true}
+      launch_vehicle_interface:=${LAUNCH_VEHICLE_INTERFACE}
       raw_vehicle_cmd_converter_param_path:=/opt/autoware/share/autoware_launch/config/vehicle/raw_vehicle_cmd_converter/raw_vehicle_cmd_converter.param.yaml
-      sensor_model:=${SENSOR_MODEL:-sample_sensor_kit}
-      use_sim_time:=${USE_SIM_TIME:-true}
-      vehicle_id:=${VEHICLE_ID:-default}
-      vehicle_model:=${VEHICLE_MODEL:-sample_vehicle}
-    restart: unless-stopped
+      sensor_model:=${SENSOR_MODEL}
+      use_sim_time:=${USE_SIM_TIME}
+      vehicle_id:=${VEHICLE_ID}
+      vehicle_model:=${VEHICLE_MODEL}
+    restart: on-failure
 
   system:
     image: ghcr.io/autowarefoundation/autoware:universe-vehicle-system
     container_name: autoware-system
     pid: service:map
+    ipc: host
     init: true
     environment:
-      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID:-0}
+      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
     networks:
       - autoware-net
     depends_on:
       - map
     command: >
       ros2 launch autoware_launch tier4_system_component.launch.xml
-      launch_system_monitor:=${LAUNCH_SYSTEM_MONITOR:-false}
-      launch_dummy_diag_publisher:=${LAUNCH_DUMMY_DIAG_PUBLISHER:-true}
-      sensor_model:=${SENSOR_MODEL:-sample_sensor_kit}
-      system_run_mode:=${SYSTEM_RUN_MODE:-planning_simulation}
-      use_sim_time:=${USE_SIM_TIME:-true}
-      vehicle_model:=${VEHICLE_MODEL:-sample_vehicle}
-    restart: unless-stopped
+      launch_system_monitor:=${LAUNCH_SYSTEM_MONITOR}
+      launch_dummy_diag_publisher:=${LAUNCH_DUMMY_DIAG_PUBLISHER}
+      sensor_model:=${SENSOR_MODEL}
+      system_run_mode:=${SYSTEM_RUN_MODE}
+      use_sim_time:=${USE_SIM_TIME}
+      vehicle_model:=${VEHICLE_MODEL}
+    restart: on-failure
+
+  sensing-perception:
+    image: ghcr.io/autowarefoundation/autoware:universe-sensing-perception
+    container_name: autoware-sensing-perception
+    pid: service:map
+    ipc: host
+    init: true
+    environment:
+      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
```

DATA_PATH is the path to autoware_data which contains NN models. This should be mounted from host.

```suggestion
    environment:
      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
    volumes:
      - ${DATA_PATH}:/autoware_data:ro
```